pom.xml

```diff
@@ -158,6 +158,11 @@
             <artifactId>jackson-annotations</artifactId>
             <version>${jackson.version}</version>
         </dependency>
+        <dependency>
```

Comment:
This needed to get added, as before it seems that `ObjectMapper` was pulled in as a transitive dependency of the old Avro version.  This dependency should have been added in the first place.